### ISSUE TITLE
[Bug]: Ark Survival Evolved egg crashes randomly after server startup

### ISSUE THREAD

**Issue by TjadenWright** [QuintenQVD0/Q_eggs#121]
### Panel Version

1.11.7

### Wings Version

1.11.7

### Service

ark survival evolved

### Modified

Yes, I modified the egg (will provide details below)

### Expected Behavior

Load ark survival evolved server and it runs without crashing. (This is on an oracle cloud vm, which is said to be support on the main page of this github).

### Actual Behavior

The server will crash with error signal 11 after a random amount of time. I have seen it from 30 min to 2-3 hours after the server fully boots. Sometimes it lasts 10-15+ hours without crashing. It's very random.

This is the error it produces. It happens a random amount of time after the "Waiting commands for 127.0.0.1:27020 (or type :q to exit)" when the server starts: 
> Signal 11 caught.
EngineCrashHandler: Signal=11
Signal 11 caught.
EngineCrashHandler: Signal=11
NativeBT: ./ShooterGameServer() [0x34a26cbc]
NativeBT: linux-vdso.so.1(__kernel_rt_sigreturn+0) [0xe8cd8455c9d0]
NativeBT: /lib/aarch64-linux-gnu/libc.so.6(gsignal+0xdc) [0xe8cd843ddeac]
NativeBT: /lib/aarch64-linux-gnu/libc.so.6(abort+0x108) [0xe8cd843caaa0]
NativeBT: ./ShooterGameServer() [0x34a0e350]
NativeBT: ./ShooterGameServer() [0x3503f398]
NativeBT: ./ShooterGameServer() [0x34a0ded4]
NativeBT: ./ShooterGameServer() [0x34a251dc]
NativeBT: ./ShooterGameServer() [0x34a25d68]
NativeBT: ./ShooterGameServer() [0x34a268f4]
NativeBT: linux-vdso.so.1(__kernel_rt_sigreturn+0) [0xe8cd8455c9d0]
NativeBT: ./ShooterGameServer() [0x3503ff24]
NativeBT: ./ShooterGameServer() [0x34a0ded4]
NativeBT: ./ShooterGameServer() [0x34a251dc]
NativeBT: ./ShooterGameServer() [0x34a25d68]
NativeBT: ./ShooterGameServer() [0x34a268f4]
NativeBT: linux-vdso.so.1(__kernel_rt_sigreturn+0) [0xe8cd8455c9d0]
NativeBT: [0xe8cb4062b600]
EmulatedBT: box64(abort+0) [0x60080]
EmulatedBT: /home/container/ShooterGame/Binaries/Linux/ShooterGameServer+16aa477 [0x1aaa477]
EmulatedBT: /home/container/ShooterGame/Binaries/Linux/ShooterGameServer+172849b [0x1b2849b]
EmulatedBT: /home/container/ShooterGame/Binaries/Linux/ShooterGameServer+1728814 [0x1b28814]
EmulatedBT: ??? [0xb]
0085|SIGABRT @0xe8cd843ddeac (gsignal(/lib/aarch64-linux-gnu/libc.so.6)) (x64pc=0x60093/"???", rsp=0xe8cd4cf71128, stack=0xe8cd4cd7b000:0xe8cd4cf7b000 own=0xe8cd4cd7b000 fp=0xe8cd4cf71130), for accessing 0x3e60000001f (code=-6/prot=0), db=(nil)((nil):(nil)/(nil):(nil)/???:clean, hash:0/0) handler=(nil)
RSP-0x20:0x0000000001ab9fa0 RSP-0x18:0x0000000000000000 RSP-0x10:0x000000003b537bb0 RSP-0x08:0x00000000000001fb
RSP+0x00:0x0000000001aaa477 RSP+0x08:0x0000e8cd4cf71240 RSP+0x10:0x0000000001b2849b RSP+0x18:0x0000000000000000
RAX:0x0000000000000000 RCX:0x000000005ba02e80 RDX:0x0000e8cd82998e80 RBX:0x0000000001ab9fa0 
RSP:0x0000e8cd4cf71128 RBP:0x0000e8cd4cf71130 RSI:0x0000e8cd82998e80 RDI:0x0000000000000001 
 R8:0x000000005ba00000  R9:0x0000000000000013 R10:0x0000000100001fff R11:0x0000000000001000 
R12:0x0000000000000000 R13:0x00000000043e911c R14:0x0000e8cd4cf71258 R15:0x000000000000001f 
ES:0x002b CS:0x0033 SS:0x002b DS:0x002b FS:0x0043 GS:0x0053 
/entrypoint.sh: line 56:    31 Aborted                 (core dumped) ./ShooterGameServer ${SERVER_MAP}?listen?SessionName="${SESSION_NAME}"?ServerPassword=${ARK_PASSWORD}?ServerAdminPassword=${ARK_ADMIN_PASSWORD}?Port=${SERVER_PORT}?RCONPort=${RCON_PORT}?QueryPort=${QUERY_PORT}?RCONEnabled=True$( [ "$BATTLE_EYE" == "1" ] || printf %s ' -NoBattlEye' ) -server ${ARGS} -log

### Steps To Reproduce

Step 1 install the latest egg from this github and import that to your pterodactyl panel.
Step 2 install steamcmd to your main os.
Step 3 In allocation create the 4 different port allocations: 7777, 7778, 27015, and 27020.
Step 4 install the server changing basic settings like passwords, server name, and the addition of the other 3 ports.
Step 5 wait for the install to complete and load the server.
Step 6 start the server and wait after the server is loaded for a random amount of time till it crashes with signal 11.

### Install logs

 https://pteropaste.com/0lab

**Comment by QuintenQVD0** [comment]
Yes I know this. This egg needs to become writen to emulate with fex and not box64
